src/main/java/core/basesyntax/FootballMatch.java

```diff
@@ -15,6 +15,11 @@
      * если не угадал ничего - возвращать 0.</p>
      */
     public int matchResult(int result1, int result2, int bet1, int bet2) {
-        return 0;
+        boolean firstTeamWin = result1 > result2 ? true : false;
```

Comment:
Can we implemented this method without redundant variables (firstTeamWin, userBetOnFirstTeam, draw, userBetOnDraw)?